src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -69,8 +71,12 @@ import {
 import { updateIssuePeriodAction } from 'common/actions/issue.actions';
 import { showAccountModalAction } from 'common/actions/general.actions';
 import { ReactComponent as BitcoinLogoIcon } from 'assets/img/bitcoin-logo.svg';
+import Vaults from '../../../components/Vaults';
```

Comment:
I put it into `components` folder, because `Chains` and `Accounts` components are very similar select components and are placed in this folder too. @tomjeatt What do you think about it?

Reply:
@peterslany cc @tomjeatt 
I have no strong opinion because it's related to convention and preference.
If you ask me, `Chains` and `Accounts` should go into `Containers` folder. I originally bootstrapped the project folder structure from that perspective.

Reply:
@crypto-engineer @peterslany - it's a good point about components not managing application state, but I don't think containers is right either (which is why Chains and Accounts are where they are). This can be left for now, but we'll be discussing the application structure during the next sprint so we can decide where these should sit then and move as appropriate in a separate PR.